piwebapi_samples/AngularJS/README.md

```diff
@@ -45,7 +45,7 @@ var testConfig = {
 };
 ```
 
-- In the terminal, use the following command to run the tests: `karma start`
+- In the terminal, use the following command to run the tests: `npx karma start`
```

Comment:
Where did this come from? Your PR suggests this is what we have in `azure-pipelines.yml`, but what I see is `npm run test-single-run`, which calls `npm test -- --single-run`, which calls `karma start karma.conf.js`
With that being said, using `npx` in the README probably is better anyway.

Reply:
Turns out there's a slight difference between our internal azure-pipelines.yml and the one on GitHub. We have a work item logged to make them match.

Reply:
OK, that makes sense. I don't have a problem with this change as-is either way, but glad we're planning to address the differences.